shell/platform/darwin/ios/ios_external_view_embedder.mm

```diff
@@ -84,7 +84,7 @@
 void IOSExternalViewEmbedder::EndFrame(bool should_resubmit_frame,
                                        fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {
   TRACE_EVENT0("flutter", "IOSExternalViewEmbedder::EndFrame");
-  FML_CHECK(platform_views_controller_);
```

Comment:
why is this deleted?

Reply:
This is a CHECK, so it is essentially crashing the app if `platform_views_controller_` is null, which is exactly what the next will do. 

I think this check shouldn't be here at all.

Reply:
I don't think there's any reason it would happen in the current code, but it might happen if rasterizer's logic changes.

Good point to have a failing message, I will add the check back and add a failing message.

Reply:
I thought about it but not sure what additional message to add that would be useful. We assume `platform_views_controller_` is not null here and it crashes if it is null. 

I could add something like "platform_views_controller_ must not be null" but it is redundant as crashing at `latform_views_controller_->EndFrame(should_resubmit_frame, raster_thread_merger);` would self-explained it.